shelldocs/src/main/python/shelldocs.py

```diff
@@ -261,12 +261,12 @@ def lint(self):
         getfuncs = {
             "audience": self.getaudience,
             "stability": self.getstability,
-            "replaceable": self.replacetext,
+            "replaceable": self.getreplace,
         }
         validvalues = {
             "audience": ("Public", "Private"),
             "stability": ("Stable", "Evolving"),
-            "replaceable": ("yes", "no"),
+            "replaceable": ("Yes", "No"),
```

Comment:
'Yes' and 'No' are not documented as being valid.

Reply:
https://github.com/apache/yetus/blob/227c762a0154c3465923d3b59ebdb3063bbd95ca/shelldocs/src/main/python/shelldocs.py#L174-L178
Here `self.getreplace` returns "Yes" or "No", therefore I changed the valid values correspondingly to pass the `elif` statement below:
https://github.com/apache/yetus/blob/227c762a0154c3465923d3b59ebdb3063bbd95ca/shelldocs/src/main/python/shelldocs.py#L278-L283

Should I update `self.getreplace` to return "yes/no" instead of "Yes/No"?

Reply:
Rethinking this, just adding a function to get the raw replaceable text is fine.

Reply:
The point of YETUS-1089 was to better enforce the ruleset and provide valid output.  'yes' or 'no' are the values that are considered valid. A value of 'Yes' or 'No' would be considered invalid and they should throw an error. Checking the validity of the rendered output rather than the parsed value doesn't do that.

Reply:
Thank you @aw-was-here for the detail. I've updated the patch to check the parsed value. 0e3b97a

Reply:
https://github.com/apache/yetus/blob/227c762a0154c3465923d3b59ebdb3063bbd95ca/shelldocs/src/main/python/shelldocs.py#L283
You said the validity check should be case-sensitive, so I thought `value.lower()` should be `value` in the error message. What do you think?

Reply:
> I thought value.lower() should be value in the error message.

Umm. It is not good for other attrs because the validity check of the attrs looks case-insensitive.